lib/xcode/install.rb

```diff
@@ -283,8 +291,8 @@ def install_dmg(dmg_path, suffix = '', switch = true, clean = true)
     end
 
     # rubocop:disable Metrics/ParameterLists
-    def install_version(version, switch = true, clean = true, install = true, progress = true, url = nil, show_release_notes = true, progress_block = nil, retry_download_count = 3)
-      dmg_path = get_dmg(version, progress, url, progress_block, retry_download_count)
+    def install_version(version, switch = true, clean = true, install = true, progress = true, url = nil, show_release_notes = true, progress_block = nil, number_of_try = 3)
+      dmg_path = get_dmg(version, progress, url, progress_block, number_of_try)
       fail Informative, "Failed to download Xcode #{version}." if dmg_path.nil?
```

Comment:
I’m actually receiving an error here. I tried with Xcode 12.2 Beta and 11.7 on this branch. I then checked out master and everything worked fine 🤔

Reply:
We didn't touch the installation part of the code. What error did you get?

Reply:
btw, how did you downloaded the 15gig dmg in 3 tries :) on master. Can trying more times (like 30) corrupt the dmg package?

Reply:
@gunesmes I actually never have any issues downloading them for some reason 🤷‍♂️ It’s nothing with the download itself because it downloaded it with this branch and it wouldn’t install. I then switched to master and then it installed the already downloaded version without an issue. I try to look into I guess since I’m the one experiencing the issue 🙃